tests/baselines/reference/nodeColonModuleResolution.trace.json

```diff
@@ -1,4 +1,5 @@
 [
+    "Module 'ph' was resolved as locally declared ambient module in file '/a/b/node_modules/@types/node/ph.d.ts'.",
```

Comment:
This is something to do with how we compile the program twice. which i dont understand clearly but when i run the scenario as "tsc" this trace is expected and was missing before.